sphinx/util/logging.py

```diff
@@ -173,7 +173,8 @@ def warning(  # type: ignore[override]
                 or the docutils node object.
             :param nonl: whether to append a new line terminator to the message
             :param color: a color code for the message
-            :param once: show message with the same ``msg`` and ``args`` only once
+            :param once: Do not log this warning,
+                if a previous warning already has same ``msg``, ``args`` and ``once=True``.
```

Comment:
Since you changed the docstring itself, could you normalize the whole docstring?

Like using a capitalized text + period for field, and perhaps use the same indentation for multiline docstring field (I would have aligned "if" with the "Do" possibly putting the if on the same line as the "warning" and making a two lines text with equal line length, but I'm not sure how we do it elsewhere).

Reply:
> Like using a capitalized text + period for field

done

> perhaps use the same indentation for multiline docstring field

they all have the same indentation (4 spaces), I feel this is good practice for aligning field lists, because aligning with the start of the first line of text means (a) you have to have an impractically large indent, (b) the indent is not consisstent across the whole list